hello-security/db-init.sql

```diff
@@ -0,0 +1,22 @@
+CREATE DATABASE IF NOT EXISTS `hello-security`;
+
+use `hello-security`;
+
+create table Person (email varchar(255), firstname varchar(255), password varchar(255), role varchar(255));
```

Comment:
It is a good idea in future to keep consistent in naming. Usually it is lowercase_with_underscores. Some more on the topic: https://medium.com/@centizennationwide/mysql-naming-conventions-e3a6f6219efe